QA/PoroDevTest/src/test/java/api_test/negative_tests/Runtime.java

```diff
@@ -0,0 +1,130 @@
+    private final FileControlUtil file = new FileControlUtil(FileControlUtil.END_TO_END_PROPERTIES);
+
+    public Runtime() throws IOException {
+    }
+
+
+
+
+
+    // Sending the runtime request with invalid token (header part with invalid list)
+    @Test(dataProvider = "invalidJwtHeaderTokenList", dataProviderClass = DataProviderBeUtil.class,
+    description = "Sending the runtime request with invalid Header part of token")
+    public void runtimeWithInvalidJwtHeaderTokenList(String invalidJwtHeaderTokenList) {
+        given().relaxedHTTPSValidation()
+                .header("authorization", "Bearer"
+                        + invalidJwtHeaderTokenList
+                        + UserDetailsGenerator.incorectJwtPayload
+                        + UserDetailsGenerator.incorectJwtSignature)
+                .when()
+                .body(UserDetailsGenerator.createRuntimeJsonReq(
+                        "e06246ba-d280-48bd-ab60-1739cee98c74",
+                        "1",
+                        "2"))
+                .post(Endpoints.RUNTIME)
+                .then()
+                .statusCode(anyOf(is(400), is(500), is(401)));
+    }
+
+
+    // Sending the runtime request with invalid token (payload part with invalid list)
+    @Test(dataProvider = "invalidJwtPayloadTokenList", dataProviderClass = DataProviderBeUtil.class,
+    description = "Sending the runtime request with invalid Payload part of token")
+    public void runtimeWithInvalidJwtPayloadTokenList(String invalidJwtPayloadTokenList) {
+        given().relaxedHTTPSValidation()
+                .header("authorization", "Bearer"
+                        + UserDetailsGenerator.validJwtHeader
+                        + invalidJwtPayloadTokenList
+                        + UserDetailsGenerator.incorectJwtSignature)
+                .when()
+                .body(UserDetailsGenerator.createRuntimeJsonReq(
+                        "e06246ba-d280-48bd-ab60-1739cee98c74",
+                        "3",
+                        "3"))
+                .post(Endpoints.RUNTIME)
+                .then()
+                .statusCode(anyOf(is(400), is(404), is(401), is(500)));
+
+    }
+
+
+
+    // Sending the runtime request with invalid token (signature part with invalid list)
+    @Test(dataProvider = "invalidJwtSignatureTokenList", dataProviderClass = DataProviderBeUtil.class,
+    description = "Sending the runtime request with invalid Signature part of token")
+    public void runtimeWithInvalidJwtSignatureTokenList(String invalidJwtSignatureTokenList) {
+        given().relaxedHTTPSValidation()
+                .header("authorization", "Bearer"
+                        + UserDetailsGenerator.validJwtHeader
+                        + UserDetailsGenerator.incorectJwtPayload
+                        + invalidJwtSignatureTokenList)
+                .when()
+                .body(UserDetailsGenerator.createRuntimeJsonReq(
+                        "e06246ba-d280-48bd-ab60-1739cee98c74",
```

Comment:
those should be extracted as separate variables

Reply:
Defied all of this attributes as final.